crtc.cpp

```diff
@@ -28,8 +28,10 @@ bool CRTControllerManager::applyConfiguration(CRTControllerManager::DockState st
     /*
      * Step 0: Check config files
      * Step 1: Open the right config file
-     * Step 3: Apply the crtc configuration
-     * Step 4: Apply the screen configuration
+     * Step 2: Parse the crtc configuration
+     * Step 3: Disable all crts
+     * Step 4: Resize the screen
```

Comment:
The screen size must be set AFTER altering the crtc config in any way. 

So disable configs, set screen size, enable configs, set screen size